webapp/src/components/views/BackendProbe.tsx

```diff
@@ -84,14 +84,14 @@ export const BackendProbe: FC<IData> = ({ onBackendFound }) => {
                 </div>
             ) : (
                 <div className={classes.informativeView}>
-                    <Title3>Looking for your backend</Title3>
+                    <Title3>Connecting...</Title3>
```

Comment:
Ideally, this page should not even exist and/or should display relevant info when maintenance mode is on!!!